prow/plugins/config.go

```diff
@@ -1076,6 +1080,26 @@ func validateTrigger(triggers []Trigger) error {
 	return nil
 }
 
+var (
+	warnSlackMergeWarningWhiteList       time.Time
+	warnSlackMergeWarningBranchWhiteList time.Time
+)
+
+func validateSlack(slack Slack) error {
+	for i := range slack.MergeWarnings {
+		if len(slack.MergeWarnings[i].WhiteList) > 0 {
+			warnDeprecated(&warnSlackMergeWarningWhiteList, 5*time.Minute, "whitelist field in Slack merge warning is deprecated and has been renamed to exempt_users. Please update your configuration.")
+			slack.MergeWarnings[i].ExemptUsers = slack.MergeWarnings[i].WhiteList
```

Comment:
Can you also error if both ExemptUsers/Whitelist or BranchWhiteList/ExemptBranches are set?

Reply:
Done in d62fdca